### DISCUSSION TITLE
Are there any plans to add time with time zone to the official documentation?

### DISCUSSION THREAD

**Discussion by wypb** [duckdb/duckdb#9335] [General]
DuckDB has actually implemented the TIME_Z type in the code([https://github.com/duckdb/duckdb/blob/main/src/include/duckdb/common/types.hpp#L212](https://www.google.com/url?q=https://github.com/duckdb/duckdb/blob/main/src/include/duckdb/common/types.hpp%23L212&sa=D&source=docs&ust=1697166304940180&usg=AOvVaw2Zk3JGQ8ny6GQnNJyO4k-J) and 
[https://github.com/duckdb/duckdb/blob/main/src/include/duckdb/common/types/datetime.hpp#L89](https://www.google.com/url?q=https://github.com/duckdb/duckdb/blob/main/src/include/duckdb/common/types/datetime.hpp%23L89&sa=D&source=docs&ust=1697166304940283&usg=AOvVaw3Gw74mD-yfFcyq3XZLbn6e)), but I didn't find it in the official [documentation](https://duckdb.org/docs/sql/data_types/overview.html). Is there any plan to add it?

**Comment by carlopi** [comment]
Can you open an issue to [duckdb/duckdb-web](https://github.com/duckdb/duckdb-web/issues) ? Should make easier to track documentation work to be done.

Thanks!

**Reply by wypb** [reply]
Hi @carlopi Thank you for your reply, I 've already created one: https://github.com/duckdb/duckdb-web/issues/1365

**Reply by carlopi** [reply]
Thanks a lot, very helpful.
